jdbc/core/src/main/java/org/apache/shardingsphere/driver/jdbc/core/driver/ShardingSphereDriverURL.java

```diff
@@ -72,4 +72,30 @@ public byte[] toConfigurationBytes() {
             return builder.toString().getBytes(StandardCharsets.UTF_8);
         }
     }
+    
+    private InputStream getResourceAsStream(final String resource) throws IOException {
+        ClassLoader[] classLoader = new ClassLoader[]{
+                Thread.currentThread().getContextClassLoader(),
+                getClass().getClassLoader(),
+                ClassLoader.getSystemClassLoader(),
+        };
+        
+        for (ClassLoader cl : classLoader) {
+            if (null != cl) {
+                
+                // try to find the resource as passed
+                InputStream returnValue = cl.getResourceAsStream(resource);
+                
+                // now, some class loaders want this leading "/", so we'll add it and try again if we didn't find the resource
+                if (null == returnValue) {
+                    returnValue = cl.getResourceAsStream("/" + resource);
+                }
```

Comment:
Did you test which class loader need '/'?
If the most frequent used one need '/', could we try to load with '/' firstly?

Reply:
`getClass().getClassLoader()` needs `/`

Reply:
sorry, I think it is not correctly. It looks like when using `getClass().getClassLoader()`, it always uses the relative path. But if there is a custom classloader, it could need the leading of `/`.  Perhaps in case of some application containers, such as `Tomcat`